iceoryx_utils/source/posix_wrapper/unix_domain_socket.cpp

```diff
@@ -412,112 +444,90 @@ cxx::error<IpcChannelError> UnixDomainSocket::createErrorFromErrnum(const int32_
     {
     case EACCES:
     {
-        std::cerr << "permission to create unix domain socket denied \"" << m_name << "\"" << std::endl;
```

Comment:
Please leave this error messages in here since they are maybe becoming valuable later.
SmartC will return only a generic error message which explains for instance `EACCES` but does not provide context for the user - in this case that it was used for socket creation.

SmartC always provides you with a rough idea and is always good to provide an additional error message to give the user a more detailed context.

Reply:
Well, I came to find that quite noisy for the user. The user is notified via `expected` about `IpcChannelError` and hence the reason why the call failed and can decide to print an info on a higher level. I could agree to just remove the print for the `ENOENT`. This is what the whole pull request is about :monkey: